src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -0,0 +1,108 @@
+import { Button, Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import BasicInfo from "pages/Admin/BasicInfo";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueDetails from "pages/Admin/Venue/Details";
+
+import { Venue_v2 } from "types/venues";
+
+import "./AdminVenueView.scss";
+
+export interface SidebarOption {
+  id: string;
+  text: string;
+}
+
+enum SidebarOptions {
+  dashboard = "dashboard",
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+  ticketingAndAccess = "ticketing_and_access",
+}
+
+const sidebarOptions: SidebarOption[] = [
+  {
+    id: SidebarOptions.basicInfo,
+    text: "Start",
+  },
+  {
+    id: SidebarOptions.entranceExperience,
+    text: "Entrance",
+  },
+  {
+    id: SidebarOptions.advancedMapSettings,
+    text: "Advanced",
+  },
+  {
+    id: SidebarOptions.dashboard,
+    text: "Dashboard",
+  },
+  // TODO: Reintroduce when field is decided what to include
+  // {
+  //   id: SidebarOptions.ticketingAndAccess,
+  //   text: "Ticketing and access",
+  // },
+];
+
+const DEFAULT_TAB = sidebarOptions.findIndex(
+  (option) => option.id === SidebarOptions.dashboard
+);
+
+interface AdminVenueViewProps {
+  venue: Venue_v2;
+}
+
+export const AdminVenueView: React.FC<AdminVenueViewProps> = ({ venue }) => {
+  const [selectedOption, setSelectedOption] = useState(
+    sidebarOptions[DEFAULT_TAB].id
+  );
+
+  const selectDashboard = useCallback(() => {
+    setSelectedOption(sidebarOptions[DEFAULT_TAB].id);
+  }, []);
+
+  return (
+    <>
+      <div className="venue-view">
+        <Button as={Link} to="/admin_v2/venue">
+          Back
+        </Button>
+
+        <Nav
+          className="venue-view__options"
+          activeKey={selectedOption}
+          onSelect={setSelectedOption}
+        >
+          {sidebarOptions.map((option: SidebarOption) => (
+            <Nav.Link
+              key={option.id}
+              className={classNames("venue-view__options--tab", {
+                selected: selectedOption === option.id,
+              })}
+              eventKey={option.id}
+            >
+              {option.text}
+            </Nav.Link>
+          ))}
+        </Nav>
+      </div>
+      {selectedOption === SidebarOptions.basicInfo && (
+        <BasicInfo venue={venue} onSave={selectDashboard} />
+      )}
+      {selectedOption === SidebarOptions.entranceExperience && (
+        <EntranceExperience venue={venue} onSave={selectDashboard} />
+      )}
+      {selectedOption === SidebarOptions.advancedMapSettings && (
+        <AdvancedSettings venue={venue} onSave={selectDashboard} />
+      )}
+      {selectedOption === SidebarOptions.dashboard && (
+        <VenueDetails venue={venue} onSave={selectDashboard} />
+      )}
```

Comment:
I suppose you are referring to this: https://github.com/sparkletown/sparkle/pull/1251#discussion_r589928518

I like the idea and I think that's the right approach. However, this means that we need to move the state in redux or pass the venueId as parameter when navigating to that route. Keep in mind that this component is also going to be responsible for venue creation in my next PR.

Reply:
> I suppose you are referring to this: #1251 (comment)

Yup, that's the one!

> I like the idea and I think that's the right approach. However, this means that we need to move the state in redux or pass the venueId as parameter when navigating to that route.

Is it just the `venueId` that will need to be in the 'state'? If so, I definitely think designing the URL 'layout' to include the `:venueId` here is the correct approach. In a similar way that we would do if we were designing API endpoints.

How/where is `AdminVenueView` rendered currently out of curiosity (specifically, on what endpoint(s)), and where will it be rendered in your followup venue creation PR?

Reply:
(additionally.. if we feel that this will end up being dramatically more complex to implement now, so long as we keep it in mind as our 'desired path' when designing/laying things out, it should be fine to 'do it in future'. The router is really cool though, as we can actually selectively render different 'sections' of a page based on the routes as well, so it can do a LOT more than what we are currently using it for with out 'change the whole page' approach)

Reply:
> How/where is AdminVenueView rendered currently out of curiosity (specifically, on what endpoint(s)), and where will it be rendered in your followup venue creation PR?

It is currently rendered conditionally in the admin v2, but I've moved it to it's own route this PR https://github.com/sparkletown/sparkle/pull/1253 and it's using `venueId` as a optional parameter in the url.

Reply:
> additionally.. if we feel that this will end up being dramatically more complex to implement now

I think it kind of is, in this PR specifically. But it won't be a huge problem to be added in my next PR: https://github.com/sparkletown/sparkle/pull/1253

Reply:
Cool cool, sounds good.